models/articles.model.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../db/connection')
+
+exports.getArticlesByIDMod = (article_id, sort_by) => {
+    const validSortBy = ['author']
+    if (sort_by && !validSortBy.includes(sort_by)) {
+        return Promise.reject({ status: 400, msg: 'bad request'})
+    }
```

Comment:
I'm not sure why there is a `sort_by` query as we haven't specified it for this ticket. It is also untested.